src/particles/plasma/PlasmaParticleContainer.cpp

```diff
@@ -58,14 +58,12 @@ PlasmaParticleContainer::ReadParameters ()
     queryWithParser(pp, "can_ionize", m_can_ionize);
     if(m_can_ionize) {
         m_neutralize_background = false; // change default
-        AMREX_ALWAYS_ASSERT_WITH_MESSAGE(!Hipace::GetInstance().m_normalized_units,
-            "Cannot use Ionization Module in normalized units");
         AMREX_ALWAYS_ASSERT_WITH_MESSAGE(m_init_ion_lev >= 0,
-            "The initial Ion level must be specified");
+            "The initial ion level must be specified");
     }
     queryWithParserAlt(pp, "neutralize_background", m_neutralize_background, pp_alt);
     AMREX_ALWAYS_ASSERT_WITH_MESSAGE(!m_can_ionize || !m_neutralize_background,
-        "Cannot use neutralize_background for Ion plasma");
+        "Cannot use neutralize_background for ion plasma");
```

Comment:
Hm is this an accurate message? I know that's not coming from this PR.
```suggestion
        "Cannot use neutralize_background when ionization is turned on");
```